app/controllers/admins_controller.rb

```diff
@@ -20,12 +21,15 @@ def new
   def edit
   end
 
+  def edit_custom
```

Comment:
Why this action is called `edit_custom` . Should be `edit` ?

Reply:
addition action edit_custom is removed.